lib/trento/application/read_models/cluster_read_model.ex

```diff
@@ -8,6 +8,7 @@ defmodule Trento.ClusterReadModel do
   import Ecto.Changeset
 
   require Trento.Domain.Enum.Provider, as: Provider
+  require Trento.Domain.Enum.Health, as: Health
```

Comment:
`Trento.Domain.Enums.Provider`
`Trento.Domain.Enums.Health`

Reply:
@rtorrero let' s fix this in another PR

Reply:
aight!